utils/modularize.js

```diff
@@ -34,9 +34,7 @@ var files = [
 	{ path: 'effects/PeppersGhostEffect.js', dependencies: [], ignoreList: [] },
 	{ path: 'effects/StereoEffect.js', dependencies: [], ignoreList: [] },
 
-	{ path: 'exporters/ColladaExporter.js', dependencies: [], ignoreList: [ 'BufferGeometry' ] },
-	{ path: 'exporters/DRACOExporter.js', dependencies: [], ignoreList: [ 'Geometry', 'BufferGeometry' ] },
```

Comment:
Is this intentional?

Reply:
No. Sorry, it's probably my misoperation done while resolving the conflicts. I'll fix.

Reply:
Done